LayoutTests/webgl/offscreen-webgl-errors-to-console.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html><!-- webkit-test-runner [ UseGPUProcessForWebGLEnabled=true, OffscreenCanvasEnabled=true ] -->
+<html>
+<body>
+<script src="../resources/js-test-pre.js"></script>
+<script>
+// Tests that using offscreen canvas WebGL, we log a warning to the console if the setting is enabled.
```

Comment:
"For tests that use offscreen canvas WebGL, we [...]"

Reply:
Not a native english speaker, but that means different thing.

What your suggestion is saying: "For red shirts that are dirty we use washing machine", e.g. it states the universal fact of the engine, qualified to subset of tests redundantly. We use washing machine for all shirts, we log console messages for all webgl tests.

What mine intended to communicate is
"[This test] Tests that using offscreen canvas WebGL, we log a warning to the console if the setting is enabled."